### ISSUE TITLE
max 100 contacts

### ISSUE THREAD

**Issue by lwedmo** [k9mail/k-9#939]
I created a contact group of 140 members for my party invitation.
I want to a send an email message to all the members of this group.
Contact group selection is capped at 100 entries.  Maybe there is a good reason for this, or maybe it's arbitrary.
As a work around, I selected the first 100, then individually hand selected the last 40 and joined the resulting addresses manually.
Can me make that cap value a settings parameter?
Or can we create a new function to copy the email addresses of a group to the clipboard?
Or we would make it possible to select a range of contacts so I don't need to check for example 40 boxes individually.
In short, contact groups don't scale well yet.

**Comment by cketti** [comment]
K-9 Mail has no support for contact groups. Whatever you're using to select the contacts seems to have this limit of 100 contacts, not K-9 Mail.
